applications/crossbar/src/modules/cb_websockets.erl

```diff
@@ -0,0 +1,244 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2011-2014, 2600Hz INC
+%%% @doc
+%%%
+%%% @end
+%%% @contributors:
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(cb_websockets).
+
+-export([init/0
+         ,authenticate/1
+         ,authorize/1
+         ,allowed_methods/0, allowed_methods/1
+         ,resource_exists/0, resource_exists/1
+         ,validate/1, validate/2
+        ]).
+
+-include("../crossbar.hrl").
+
+-define(CB_LIST, <<"websockets/crossbar_listing">>).
+-define(
+    AVAILABLE
+    ,wh_json:from_list([
+        {<<"call">>, [<<"call.CHANNEL_CREATE.*">>, <<"call.CHANNEL_ANSWER.*">>, <<"call.CHANNEL_DESTROY.*">>, <<"call.RECORD_STOP.*">>]}
```

Comment:
<<"call.RECORD_STOP.*">> is this actually supported ?
also, there was a recent addition to wh_hooks_listener for CHANNEL_BRIDGE, which gets supported here automagically.